src/main/java/uk/gov/justice/digital/hmpps/csr/api/repository/DetailRepository.kt

```diff
@@ -0,0 +1,208 @@
+package uk.gov.justice.digital.hmpps.csr.api.repository
+
+import org.springframework.jdbc.core.RowMapper
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.csr.api.model.Detail
+import java.sql.ResultSet
+import java.time.LocalDate
+
+@Repository
+class DetailRepository(private val jdbcTemplate: NamedParameterJdbcTemplate) {
+
+    fun getDetails(from: LocalDate, to: LocalDate, quantumId: String): Collection<Detail> {
+        return jdbcTemplate.query(
+                GET_DETAILS,
+                MapSqlParameterSource()
+                        .addValue("from", from)
+                        .addValue("to", to)
+                        .addValue("quantumId", quantumId),
+                detailsRowMapper
+        )
+    }
+
+    fun getModifiedShifts(planUnit: String): Collection<Detail> {
```

Comment:
what's the difference between `getModifiedShifts` and `getModifiedDetails`? It sounds like the former shouldn't be on this repository